.github/workflows/issue-bot.yaml

```diff
@@ -32,3 +32,9 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Update issue status if the issue is closed as completed
+        if: github.event.action == 'closed' && contains(github.event.issue.body, 'closed as completed')
+        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:in-progress" --add-label "status:new-release"
```

Comment:
Honestly I'm not sure if we should be using this label, I'm not sure it makes sense.

Reply:
Should `status:in-progress` just be removed then when an issue is closed on completion, leaving it with no `status` label?

Reply:
Yeah that'd make sense, `status:accepted` could be removed too?